HelloMixpanel/HelloMixpanel/AppDelegate.m

```diff
@@ -38,17 +38,32 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [self.mixpanel registerSuperProperties:@{@"Plan": @"Premium"}];
 
     // Set a profile property so a profile is created
-    [self.mixpanel.people setOnce:@{@"$name": @"Demo User"}];
+    [self.mixpanel.people setOnce:@{@"$name": @"Demo User1"}];
 
     // Track a test event
     [self.mixpanel track:@"HelloMixpanel"];
 
     // Identify using the generated distinctId so people queue is flushed
-    [self.mixpanel identify:[self.mixpanel distinctId]];
+    [self.mixpanel identify:@"testidentify"];
 
     // Force a flush to make debugging easier
     [self.mixpanel flush];
     
+    
+    NSError *e = nil;
+    NSString *jsonString = @"[{\"id\": false, \"name\":\"sam\"}]";
+    NSData *data = [jsonString dataUsingEncoding:NSUTF8StringEncoding];
+
+    NSArray *jsonArray = [NSJSONSerialization JSONObjectWithData: data options:  NSJSONReadingMutableContainers error: &e];
+
+    if (!jsonArray) {
+        NSLog(@"Error parsing JSON: %@", e);
+    } else {
+        for(NSDictionary *item in jsonArray) {
+            NSLog(@"Item: %@", item);
```

Comment:
what's the purpose for adding this code?

Reply:
whoops, will delete.